JetStreamDriver.js

```diff
@@ -508,7 +528,28 @@ class Driver {
         }
     }
 
-    resultsObject()
+    resultsObject(newStyle = false) {
+        if (newStyle)
+            return this.resultsObjectNew();
+        return this.resultsObjectOld();
```

Comment:
Old vs New might not be the best distinction since WebKit's infrastructure relies on the "old" JSON schema and I don't think it's likely to change in the foreseeable future. Can we call it something else? Unfortunately, I don't have a great name... maybe call the old thing `runBenchmarkResultsObject()` (`run-benchmark` is WebKit's benchmark harness script)?